packages/babel-plugin/src/visitors/stylex-create/parse-stylex-create-arg.js

```diff
@@ -77,9 +78,27 @@ export function evaluateStyleXCreateArg(
         pathUtils.isIdentifier(param),
       )
       .map((param) => param.node);
+
+    if (allParams.some((param) => pathUtils.isObjectPattern(param))) {
+      throw new Error(
+        messages.ONLY_NAMED_PARAMETERS_IN_DYNAMIC_STYLE_FUNCTIONS,
+      );
+    }
+
+    if (
+      allParams.some(
+        (param) =>
+          pathUtils.isAssignmentPattern(param) &&
+          t.isIdentifier(param.node.left),
```

Comment:
actually, I don't think this identifier check is necessary. any assignment is invalid